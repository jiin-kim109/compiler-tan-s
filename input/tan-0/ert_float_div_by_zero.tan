main {
    const r := 3.2 / 0.0;
    print r \n;
}