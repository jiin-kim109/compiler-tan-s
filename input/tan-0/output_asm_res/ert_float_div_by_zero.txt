Stack overflow on PushI instruction at location 10.
