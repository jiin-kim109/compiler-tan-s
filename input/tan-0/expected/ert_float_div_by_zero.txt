Runtime error: floating divide by zero
